templates/author/submit/step3.tpl

```diff
@@ -77,131 +77,134 @@ function moveAuthor(dir, authorIndex) {
-</tr>
-<tr valign="top">
-	<td width="20%" class="label">{fieldLabel name="authors-0-lastName" required="true" key="user.lastName"}</td>
-	<td width="80%" class="value"><input type="text" class="textField" name="authors[0][lastName]" id="authors-0-lastName" size="20" maxlength="90" /></td>
-</tr>
-<tr valign="top">
-	<td width="20%" class="label">{fieldLabel name="authors-0-email" required="true" key="user.email"}</td>
-	<td width="80%" class="value"><input type="text" class="textField" name="authors[0][email]" id="authors-0-email" size="30" maxlength="90" /></td>
-</tr>
-<tr valign="top">
-	<td width="20%" class="label">{fieldLabel name="authors-0-orcid" key="user.orcid"}</td>
-	<td width="80%" class="value"><input type="text" class="textField" name="authors[0][orcid]" id="authors-0-orcid" size="30" maxlength="90" /><br />{translate key="user.orcid.description"}</td>
-</tr>
-<tr valign="top">
-	<td width="20%" class="label">{fieldLabel name="authors-0-url" key="user.url"}</td>
-	<td width="80%" class="value"><input type="text" class="textField" name="authors[0][url]" id="authors-0-url" size="30" maxlength="255" /></td>
-</tr>
-<tr valign="top">
-	<td width="20%" class="label">{fieldLabel name="authors-0-affiliation" key="user.affiliation"}</td>
-	<td width="80%" class="value">
-		<textarea name="authors[0][affiliation][{$formLocale|escape}]" class="textArea" id="authors-0-affiliation" rows="5" cols="40"></textarea><br/>
-		<span class="instruct">{translate key="user.affiliation.description"}</span>
-	</td>
-</tr>
-<tr valign="top">
-	<td width="20%" class="label">{fieldLabel name="authors-0-country" key="common.country"}</td>
-	<td width="80%" class="value">
-		<select name="authors[0][country]" id="authors-0-country" class="selectMenu">
-			<option value=""></option>
-			{html_options options=$countries}
-		</select>
-	</td>
-</tr>
-{if $currentJournal->getSetting('requireAuthorCompetingInterests')}
-<tr valign="top">
-	<td width="20%" class="label">{fieldLabel name="authors-0-competingInterests" key="author.competingInterests" competingInterestGuidelinesUrl=$competingInterestGuidelinesUrl}</td>
-	<td width="80%" class="value"><textarea name="authors[0][competingInterests][{$formLocale|escape}]" class="textArea" id="authors-0-competingInterests" rows="5" cols="40"></textarea></td>
-</tr>
-{/if}
-<tr valign="top">
-	<td width="20%" class="label">{fieldLabel name="authors-0-biography" key="user.biography"}<br />{translate key="user.biography.description"}</td>
-	<td width="80%" class="value"><textarea name="authors[0][biography][{$formLocale|escape}]" class="textArea" id="authors-0-biography" rows="5" cols="40"></textarea></td>
-</tr>
-</table>
+<div class="row">
+	<div class="col-md-12">
+		<div class="label col-md-2">{fieldLabel name="authors-0-firstName" required="true" key="user.firstName"}</div>
```

Comment:
Shouldn't the .col div be always inside .row div (based on the bootstrap examples, you shouldn't nest .col inside .col without using the .row between them) ?